CHANGES.md

```diff
@@ -1,8 +1,21 @@
-4.0.0.0 2018-02-23
+4.0.1 2018-04-06
 ==================
 
--   V205 APIs are now called by default -- this is an incompatible change
-    (use version='204' to call the previous API version)
+- Updated documentation in CHANGES.md and README.md
+
+4.0.0.0 2018-03-30
+==================
+
+- Adds support for Sift Science API Version 205, including new [`$create_content`](https://siftscience.com/developers/docs/curl/events-api/reserved-events/create-content) and [`$update_content`](https://siftscience.com/developers/docs/curl/events-api/reserved-events/update-content) formats
+- V205 APIs are now called -- **this is an incompatible change**
+   - Use `version = '204'` when constructing the Client to call the previous API version
+- Adds support for content decisions to [Decisions API](https://siftscience.com/developers/docs/curl/decisions-api)
+
+
+INCOMPATIBLE CHANGES INTRODUCED IN API V205:
+- `$create_content` and `$update_content` have significantly changed, and the old format will be rejected
+- `$send_message` and `$submit_review` events are no longer valid
+- V205 improves server-side event data validation.  In V204 and earlier, server-side validation accepts some events that did not conform to the published APIs in our [developer documentation](https://siftscience.com/developers/docs/curl/events-api).  V205 does not modify existing event APIs, but may reject invalid event data that were previously accepted.  **Please test your integration on V205 in sandbox before using in production.**
```

Comment:
Let's make some small changes to this line for all clients:
`V205 improves server-side event data validation.  In V204 and earlier, server-side validation accepted some events that did not conform to the published APIs in our [developer documentation](https://siftscience.com/developers/docs/curl/events-api).  V205 does not modify existing event APIs other than those mentioned above, but may reject invalid event data that were previously accepted.  **Please test your integration on V205 in sandbox before using in production.**`

I changed "accepts" to "accepted" (to match tense) and added " other than those mentioned above"